uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -11,35 +11,32 @@ define([
   'leave-absences/shared/models/entitlement-model',
   'leave-absences/shared/models/leave-request-model',
   'leave-absences/shared/models/public-holiday-model',
-  'leave-absences/shared/models/instances/leave-request-instance',
-  'leave-absences/shared/models/instances/sickness-leave-request-instance',
-], function (components, _, moment) {
+], function (controllers, _, moment) {
```

Comment:
there are many instance of deep nested callbacks where this is required and in order to be consistent used the convention of that ->this instead of mixing bind with that--> this